cachito/web/utils.py

```diff
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 from collections import OrderedDict
+from collections.abc import Iterator
+from typing import Any, Dict, List
```

Comment:
If the `Iterator` import is only for typing, could it be imported from the `typing` modules instead?